src/com/couchbase/tools/performer/BuildDockerRubyPerformer.groovy

```diff
@@ -16,7 +16,7 @@ class BuildDockerRubyPerformer {
      * @param imageName  the name of the docker image
      * @param onlySource whether to skip the docker build
      */
-    static void build(Environment imp, String path, VersionToBuild build, String imageName, boolean onlySource = false) {
+    static void build(Environment imp, String path, VersionToBuild build, String imageName, boolean onlySource = false, String[] dockerBuildArgs = new String[0]) {
```

Comment:
Can we use something nicer than raw arrays?

Reply:
Something like `List<String>`?

Reply:
Or since they are key-value pairs it might be better to use a `Map` here?

Reply:
Done